ref/Elementary_Drawing_Functions.md

```diff
@@ -266,3 +266,32 @@ The following piece of code shows a combined usage of the `drawcircle` and the `
 | ![Image](img/FillCircleX.png) |
 | ----------------------------- |
 | ****                          |
+
+------
+
+#### Drawing circular arc: `drawarc(‹point› A,<point> B, <point> C)`
```

Comment:
Come to think of it, this distinction between type and name is unprecedented in the documentation, as far as I can tell. Other locations with multiple arguments of equal type tend to number their arguments. So you'd write `drawarc(‹point1›,‹point2›,‹point3›)`. The idea as I understand it is that `‹point1›` as a whole serves as a placeholder.

Personally I'm open to a change of that convention, but I think it should be discussed and perhaps even be applied consistently to existing documentation.